docs/src/theme/DocBreadcrumbs/Items/Home/index.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Link from "@docusaurus/Link";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { translate } from "@docusaurus/Translate";
+import IconHome from "@theme/Icon/Home";
+import styles from "./styles.module.css";
+import AztecLogo from "@site/static/img/Aztec_icon_minified.svg";
+
+export default function HomeBreadcrumbItem() {
+  const homeHref = useBaseUrl("/");
+  return (
+    <li className="breadcrumbs__item" itemProp="itemListElement">
+      <Link
+        aria-label={translate({
```

Comment:
Was this file deleted with the restructure, and this adds it back?
If not, curious how this fixes it vs how it was working previously?

Reply:
Apparently when there isn't a list of breadcrumbs, docusaurus doesn't add this `itemListElement`.

Not sure if this happens only on docusaurus v3, but we also made some changes to the sidebar so maybe it has something to do with this.

In any case swizzling the breadcrumb item was the proposed solution on one docusaurus discussion on it, and it works perfectly